helix-term/src/ui/completion.rs

```diff
@@ -88,6 +88,12 @@ impl menu::Item for CompletionItem {
     }
 }
 
+#[derive(Debug, PartialEq, Default, Clone)]
+struct CompletionItem {
+    item: lsp::CompletionItem,
+    resolved: bool,
+}
```

Comment:
Would it be better to put this as enum?

Reply:
in #5336 this will become a struct anyway (the source server needs to be stored) so I don't think that's a good fit here. Just for the resolved status alone you would endup with:

``` rust
enum ResolveStatus{
	Resolved,
	Unresolved,
}
```

which is not any clearer. Also completion item and the resolved status are orthogonal properties and we only care whether an item is resolved in a few place so I don't think it makes sense right now.